src/test/java/io/airlift/slice/TestFnvHash.java

```diff
@@ -74,11 +74,11 @@ public void testFNVHash64()
                 0x456fbb181822b8f6L
         };
 
-        assertEquals(testData.length, expected.length);
+        assertThat(testData.length).isEqualTo(expected.length);
```

Comment:
```java
assertThat(testData).hasSameSizeAs(expected);

Reply:
`hasSameSizeAs` doesn't seem to be defined for arrays